README.md

```diff
@@ -1,13 +1,13 @@
 # prebuild-install [![Build Status](https://travis-ci.org/mafintosh/prebuild-install.svg?branch=master)](https://travis-ci.org/mafintosh/prebuild-install)
 
-A command line tool for easily install prebuilds for multiple version of node/iojs on a specific platform.
+Command line tool to easily install prebuilds for multiple version of node/iojs
```

Comment:
this sentence sounded better before

Reply:
I agree on that the first sentence sounded better before, but I believe it has a grammatical error. Afaik it should be ".. for easily installing ..".